src/test/kotlin/factory/PositionFactoryTest.kt

```diff
@@ -0,0 +1,21 @@
+package factory
+
+import enum.PositionType
+import org.junit.jupiter.api.Assertions
+
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+
+class PositionFactoryTest {
+  @ParameterizedTest
```

Comment:
@ParameterizedTest(name = "BrExecutionEnvironment {arguments} exists in test execution EnvironmentType")